hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -14,25 +15,32 @@
 
 from health_ml.utils.bag_utils import BagDataset, multibag_collate
 from health_ml.utils.common_utils import _create_generator
+from health_ml.utils.wsi_utils import list_data_collate
 
-from histopathology.datasets.base_dataset import TilesDataset
+from histopathology.datasets.base_dataset import SlidesDataset, TilesDataset
+from histopathology.datasets.panda_dataset import PandaDataset
 from histopathology.models.transforms import LoadTilesBatchd
 
+from monai.transforms.compose import Compose
+from monai.transforms.io.dictionary import LoadImaged
+from monai.apps.pathology.transforms import TileOnGridd
+from monai.data.image_reader import WSIReader
+
 
 class CacheMode(Enum):
-    NONE = 'none'
-    MEMORY = 'memory'
-    DISK = 'disk'
+    NONE = "none"
+    MEMORY = "memory"
+    DISK = "disk"
 
 
 class CacheLocation(Enum):
-    NONE = 'none'
-    CPU = 'cpu'
-    SAME = 'same'
+    NONE = "none"
+    CPU = "cpu"
+    SAME = "same"
```

Comment:
Checking if your formatting setup match those of the repo? It shouldn't happen when you commit that single quote transform into double quote of the opposite. But here it might be an old file that has been untouched for a while